src/index.html

```diff
@@ -16,7 +16,107 @@
       href="./style.css"
     />
   </head>
+  <link
+    rel="preconnect"
+    href="https://fonts.googleapis.com"
+  />
+  <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+  />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+    rel="stylesheet"
+  />
```

Comment:
The 'link' tag for fonts should be inside the 'head' tag. It's a good practice to put all links and meta tags inside the 'head' tag.